libs/deal-ticket/src/components/info-market.tsx

```diff
@@ -23,97 +23,101 @@ export interface InfoProps {
 export const Info = ({ market }: InfoProps) => {
   const headerClassName =
     'text-h5 font-bold uppercase text-black dark:text-white';
+  const marketDataPanels = [
+    {
+      key: 'fees',
```

Comment:
question (non-blocking): do we need these key properties? they don't seem to be used anywhere in the code